examples/rpm/README.md

```diff
@@ -0,0 +1,18 @@
+# RPM Install Example
+
+This is a basic install using the "rpm" install method.
+
+The "tar" method is the default because it is more generic and contained; the tar method works on more OS and does not require public internet access.
```

Comment:
Tar is the default deployment method because it is portable across many operating systems, versatile, and independent of the internet.

Reply:
nice, yeah, will update.